pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -492,7 +492,7 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
     };
 
     $scope.getStayEnrolledInCurrentSection = function() {
-      if ($scope.selectedSectionCode == $scope.manuallySelectedSectionCode) {
+      if ($scope.selectedSectionCode == $scope.otherTeacher) {
```

Comment:
Note, this now matches the check that occurs [just a few lines earlier in `getNewSectionCode`](https://github.com/code-dot-org/code-dot-org/pull/10444/files#diff-ad3e28ff4c37f5f09bfae73539121a12R487), which is probably what the original author intended.